Assignment1/Assignment1_QNo1/SolAssignmentNo1_QNo1.rb

```diff
@@ -0,0 +1,25 @@
+# => 1. Find out how to convert the string "7.13" into the number 7.13
+# => author : Hrisikesh Singha Chowdhury
+
+# => definition of custom class
+class CustomClass
+  # => Fn to implement buisness logic
+  def cvt_str_to_int
+    puts "Enter another string"
+    str = gets.chomp
+    str_output = str.chomp.to_f
+
+    if (str.nil? && str_output.zero?)
+      puts "Something wrong"
+    else
+      puts "The required floating conversion is #{str_output}"
+    end
```

Comment:
@HrisikeshSinghaChowdhury What will happen if given input is some word instead of any numeric value? That is case not handled here!

Reply:
done

Reply:
@HrisikeshSinghaChowdhury Zero is a valid number

Reply:
@HrisikeshSinghaChowdhury the input is other than any numeric value that case is not yet handled in your code

Reply:
done